.github/workflows/pull-request.yml

```
@@ -17,10 +17,10 @@ jobs:
             echo "Merge requests to main branch are only allowed from develop branch."
             exit 1
           fi
-      - name: Check if PR to develop is from feature/*
+      - name: Check if PR to develop is from feature/* or bug/* or improvement/*
         if: github.event.pull_request.base.ref == 'develop'
         run: |
```

actionlint (1.7.7)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)